src/relayFeeCalculator/chain-queries/svmQuery.ts

```diff
@@ -144,9 +136,9 @@ export class SvmQuery implements QueryInterface {
       recipient: SvmAddress;
       outputToken: SvmAddress;
     },
-    relayer = toAddressType(getDefaultRelayer(relayData.destinationChainId), relayData.destinationChainId),
+    relayer = getDefaultRelayer(relayData.destinationChainId),
     repaymentChainId = relayData.destinationChainId,
-    repaymentAddress = toAddressType(getDefaultRelayer(relayData.destinationChainId), relayData.destinationChainId)
+    repaymentAddress = getDefaultRelayer(relayData.destinationChainId)
```

Comment:
nb. this is actually pretty dangerous because it permits _any_ caller to generate an SVM fill txn without specifying their repayment address.

We could mark `getFillRelayTx()` `protected` to ensure it's only used internally for gas cost estimation. Alternatively, we could make it mandatory to specify `repaymentAddress`.

Reply:
Just wanted to flag it for comments before fixing it. I updated it here: https://github.com/across-protocol/sdk/pull/1108/commits/886b894c57c1f0d81f0d63cb2f012358098349ec